.travis.yml

```diff
@@ -148,6 +148,11 @@ jobs:
       env: >-
         FILE_ENV="./ci/test/00_setup_env_native_fuzz.sh"
 
+    - stage: test
```

Comment:
not sure if this is useful to run on travis. This is an experimental feature (currently not even a feature). So the inconvenience in the unlikely case that it breaks is probably not worth having it run on every pull request. Please keep the ci config, though. If you want you can add it to `.cirrus.ci` here in the repo or people can run it manually on their machines.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/18677#discussion_r423688995

> not sure if this is useful to run on travis

What if I added an `if: commit_message !~ /depends:|multiprocess:/` [condition](https://docs.travis-ci.com/user/conditional-builds-stages-jobs)? This travis config has uncovered a few build and runtime bugs that weren't happening for me locally, so I think it's been pretty useful. The idea behind the regex would be not run on normal pull requests but catch regressions on depends and multiprocess pull requests.

Reply:
Absolutely, if you can get the condition to work. Travis is running the build on a temporary merge commit, so the commit message might not might not be the one you pushed.